devops/scripts/ensure_workspace_metadata.py

```diff
@@ -0,0 +1,77 @@
+from arango import ArangoClient
+import getpass
+import sys
+
+from mypy_extensions import TypedDict
```

Comment:
mypy_extensions are not in the Pipfile which causes errors on fresh installs. It may be worth adding it in, even as a dev dependency

Reply:
Good catch. For that matter, `mypy` isn't listed in `Pipfile` either. I will send this update on a separate branch, so as not to conflict with the content changes in this branch.

Reply:
See #399.